bears/generate_package.py

```diff
@@ -159,12 +161,23 @@ def main():
                                       substitution_dict)
 
             bear_dist_name = bear_name + '-' + bear_version
-            if args.register:
+            if args.register and not args.bear:
                 perform_register(os.path.join('bears', 'upload', bear_name),
                                  bear_dist_name)
-            if args.upload:
+            if args.upload and not args.bear:
                 perform_upload(os.path.join('bears', 'upload', bear_name))
 
+    if args.bear:
```

Comment:
i think we should be able to upload multiple bears such as running:
"-b CSSLintBear CSharpBear" or so, it would be nice.